backend/PythonClient/multirotor/airsim_application.py

```diff
@@ -6,14 +6,14 @@
 
 from PythonClient import airsim
 from google.cloud import storage
-
+from PythonClient.multirotor.monitor.gsc_storage_service import GCSStorageService
 
 class AirSimApplication:
     # Parent class for all airsim client side mission and monitors
     def __init__(self):
-        self.bucket_name = 'droneworld'  # The bucket created in GCS
-        self.storage_client = storage.Client.from_service_account_json('key.json')  # Initializes the GCS client
-        self.bucket = self.storage_client.bucket(self.bucket_name)  # Points to the GCS bucket
+        #implementation of the GCS service
+        self.gcs = GCSStorageService('droneworld')
```

Comment:
This should be self.storage_service = GCSStorageService('droneworld')

Reply:
changed